### PR TITLE
Pull in rancher-backup charts from their repository

### PR DESCRIPTION
**Pull request by nickgerace**
With this PR, there will be two packages for `rancher-backup`: the standard chart and the CRD chart. This is similar to how `rancher-operator` and `rancher-operator-crd` is packaged. The charts will be moved to [rancher/backup-restore-operator](https://github.com/rancher/backup-restore-operator/).

Relevant issues:
- https://github.com/rancher/backup-restore-operator/issues/78
- https://github.com/rancher/backup-restore-operator/issues/71
- https://github.com/rancher/backup-restore-operator/issues/72
- https://github.com/rancher/backup-restore-operator/issues/69

This PR cannot be fully evaluated until https://github.com/rancher/backup-restore-operator/pull/79 is merged.

### CODE REVIEW

**Comment by nickgerace** [comment]
The first build is expected to fail. This PR depends on the chart being moved and _released_.

**Review by aiyengar2 — CHANGES_REQUESTED** [review]
Anyone against bumping the rancher-backup version?

File: packages/rancher-backup-crd/package.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+url: https://github.com/rancher/backup-restore-operator/releases/download/v1.0.3/rancher-backup-crd-1.0.3.tgz
+packageVersion: 01
+releaseCandidateVersion: 02
```
Comment (aiyengar2) [inline packages/rancher-backup-crd/package.yaml:3]: We should bump the rancher-backup version to 1.0.4 and switch the packageVersion and releaseCandidateVersion to 00 and 00 to be in sync with other charts of a similar nature (e.g. fleet stuff today).

https://github.com/rancher/charts/blob/ead512d870063ab3164c7484795f8c9dd0f5b599/packages/fleet-agent/package.yaml#L2-L3

Comment (nickgerace) [reply packages/rancher-backup-crd/package.yaml:3]: Agreed.

**Comment by nickgerace** [comment]
Going to work on this PR again now that https://github.com/rancher/backup-restore-operator/releases/tag/v1.0.4-rc2 is released.

**Review by aiyengar2 — DISMISSED** [review]
LGTM

**Review by jiaqiluo — APPROVED** [review]
LGTM

**Pull request merged**